roboswag/auth.py

```diff
@@ -28,10 +28,24 @@ def get_from_kwargs_or_robot(kwargs, name, missing_ok=False):
 
 
 class TokenHandler:
+    # FIXME
```

Comment:
What needs to be fixed?

Reply:
It doesn't work at all - for now we just use BasicAuth. TokenHandler was in the code already and I left it for the time being but in future PR it will be refactored. Left comment to not forget this